docs/_static/nidigital_usage.inc

```diff
@@ -3,13 +3,29 @@ Usage
 
 The following is a basic example of using the **nidigital** module to open a session to a ....
 
+.. code-block:: python
+
+    import nidigital
+    with nidigital.Session(resource_name='PXI1Slot2', channels='0') as session:
+        pass
+
+
+Some repeated capabilities can be nested. This is useful for some methods that can be used with the `pins`
+repeated capability. They can be nested with the `sites` repeated capability.
+
 .. code-block:: python
 
     import nidigital
     # Configure the session.
 
     with nidigital.Session(resource_name='PXI1Slot2', channels='0') as session:
-        pass
+        session.sites[0, 1].pins[2, 3].burst_pattern(
```

Comment:
`burst_pattern` cannot be used with `pins` repeated capability. Use `ppmu_source` in the exameple:
```
    session.sites[0, 1].pins['PinA', 'PinB'].ppmu_source()
```

Reply:
Changed.